client/src/components/utils/appHelper.js

```diff
@@ -42,11 +57,24 @@ const convertPlaceIdToLatLng = place => (
   })
 );
 
+/**
+ * Check if current location is available.
+ * @param  {Object} props Props passed into the App component
+ * @return {String}       Description of current location availability
+ */
 const getCurrentLocationCardPrimaryText = props => (
   // Check that current location has been reset to a real location
   props.currentLocation.lat !== 400 ? 'Use current location' : 'Current location not available'
 );
 
+/**
+ * Save results for origin and destination searches.
+ * @param  {Array}  predictions      Collection of predicted place objects
+ * @param  {String} status           The status returned by the Google Maps Places Service
+ * @param  {Object} props            Props passed into the App component
+ * @param  {Object} interactionTypes Available immutable interaction types
+ * @return {Function Invocation}     Dispatched action
```

Comment:
Agree with David on this one. I think return here should be undefined.

I'm not sure why I even wrote a return statement for this helper, as I'm sure it would still function properly without a return.